### PR TITLE
Re-enable LDC testing + small improvement to the docs action

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Comment by Geod24** [comment]
Note: the end goal is to drop DMD. But we better ensure we have at least 2 compilers that can compile and run our code successfully.

File: .github/workflows/github_push.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
     name: MacOS
     strategy:
       matrix:
-        os: [macOS-latest]
+        os: [macOS-15.04]
```
Comment (AndrejMitrovic) [inline .github/workflows/github_push.yaml:9]: Why 15.04?

Comment (Geod24) [reply .github/workflows/github_push.yaml:9]: That's the latest version (aka 10.15.04)

Comment (AndrejMitrovic) [reply .github/workflows/github_push.yaml:9]: Oh this is Mac, not Ubuntu. My bad.

**Comment by Geod24** [comment]
So are we good to go ?

**Comment by Geod24** [comment]
Needed by #691

**Comment by TrustHenry** [comment]
looks good to me. Debugging is also good.  However, I haven't tested it on the Windows version.

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**